src/app/code/community/Meanbee/Core/Model/System/Config/Source/StaticBlock.php

```diff
@@ -0,0 +1,28 @@
+<?php
+/**
+ * Class Meanbee_Core_Model_System_Config_Source_StaticBlock
+ */
+class Meanbee_Core_Model_System_Config_Source_StaticBlock implements Meanbee_Core_Model_System_Config_SourceModel
+{
+    /**
+     * Return a source model array containing all static blocks defined in the system.
+     *
+     * @return array
+     */
+    public function toOptionArray()
+    {
+        $options = array();
+        $blocks = Mage::getModel('cms/block')->getCollection()
+            ->addFieldToFilter('is_active', 1);
+
+        foreach ($blocks as $block) {
+            /** @var Mage_Cms_Model_Block $block */
+            $options[] = array(
+                'value' => $block->getIdentifier(),
+                'label' => sprintf("%s (id: %s)", $block->getTitle(), $block->getIdentifier())
+            );
+        }
```

Comment:
I think this should have an empty option. Otherwise, it's confusing on the frontend when nothing is selected, but the select box still defaults to the first option, which might make people think it's already set (in the system configuration context).